tests/server/blueprints/institutes/test_institute_views.py

```diff
@@ -686,6 +686,17 @@ def test_cases_similar_phenotype_query(app, case_obj, institute_obj, test_hpo_te
         assert case_obj["display_name"] in str(resp.data)
 
 
+def test_verified(app, institute_obj):
+    """Test the endpoint that returns the list of verified variants for an institute"""
+    # GIVEN an initialized app
+    with app.test_client() as client:
```

Comment:
I'm not really satisfied with this test, tomorrow I could improve it to include a case where you mark a variant as validated and then you search it in the validated page